POINT-VR-Chapter-1/.gitignore

```diff
@@ -74,3 +74,6 @@ crashlytics-build.properties
 /[Pp]rojectSettings/ProjectSettings.asset
 /[Pp]rojectSettings/EditorBuildSettings.asset
 /[Pp]ackages/packages-lock.json
+
+**/EmulatorComponents/*
+EmulatorComponents.meta
```

Comment:
I was recently told that it is better to always include a new line at the end as git is a bit unhappy otherwise, so I'll add that in a commit before we merge this.